src/libs/actions/Report.js

```diff
@@ -87,6 +87,19 @@ Onyx.connect({
     },
 });
 
+const draftNoteMap = {};
```

Comment:
NAB: Maybe `privateNotesDraftMap`? So we exactly know it is something related to private notes feature.